core/templates/dev/head/pages/exploration-player-page/services/image-preloader.service.ts

```diff
@@ -63,7 +63,10 @@ angular.module('oppia').factory('ImagePreloaderService', [
           if (_isInFailedDownload(loadedImageFile.filename)) {
             _removeFromFailedDownload(loadedImageFile.filename);
           }
-          var objectUrl = URL.createObjectURL(loadedImageFile.data);
+          var objectUrl = (
+            window.URL || window.webkitURL || window ||
+            {}).createObjectURL(loadedImageFile.data);
```

Comment:
Might throw a type error if it evaluates to `{}.creteObjectURL()`. Can you check what happens if an error is thrown? If it prevents further user interaction it would be good to handle this in a try..catch (remember to throw the error in the catch so it shows up in the logs). @seanlip do you have any thoughts on this?

Reply:
Perhaps the existence of createObjectURL is something to check at the very outset, when this service is initialized. If it doesn't exist then we throw an error and stop further processing.

Reply:
Yes, you are correct @kevintab95  ``` { }.createObjectURL( )```  will throuh an error.

While related to this
> Can you check what happens if an error is thrown?
When this happens, then there is no rendering of the image. User interaction will be the infinite loop of loading an image.
**Screenshot of user interaction**
![Screenshot from 2019-08-22 04-59-56](https://user-images.githubusercontent.com/34139754/63475265-c4110180-c499-11e9-9e24-900273b00b6d.png)
**screenshot of console error**
![imageedit_2_9190492399](https://user-images.githubusercontent.com/34139754/63475338-0f2b1480-c49a-11e9-9707-e37df5762255.png)

Reply:
I agree with Sean's recommendation to throw an error when the service is initialized if createObjectURL is not available. @import-keshav could you please make that change? Thanks.